docs/workflow/testing/mono/testing.md

```diff
@@ -51,4 +51,4 @@ You can then, for example, run our HelloWorld sample via:
 dotnet build -c Release $(REPO_ROOT)/src/mono/netcore/sample/HelloWorld
 MONO_ENV_OPTIONS="" COMPlus_DebugWriteToStdErr=1 \
 $(REPO_ROOT)/.dotnet-mono/dotnet $(REPO_ROOT)/src/mono/netcore/sample/HelloWorld/bin/HelloWorld.dll
-```
```

Comment:
@CoffeeFlux GitHub doesn't show a difference. But it may be a blank line at the end of the file. Not sure.

Here is the code I used to initially generate this PR.

```
void Main()
{	
	var markdown = Directory.GetFiles(@"D:\dotnet\runtime\", "*.md", SearchOption.AllDirectories).ToArray();
	
	for (var i = 0; i < markdown.Length; i++)
	{
	    string[] lines = File.ReadAllLines(markdown[i]);
            File.WriteAllLines(markdown[i], lines.Select(s => s.TrimEnd()));
	}
}
```